core/src/eval/tests.rs

```diff
@@ -188,7 +188,7 @@ fn imports() {
         var: &str,
         import: &str,
         body: NickelValue,
-    ) -> Result<NickelValue, ImportError>
+    ) -> Result<NickelValue, ImportErrorKind>
```

Comment:
Why not box up this error variant (similar to EvalErrorKind)?

Reply:
In fact I suspect the actual reason is that `ImportErrorKind` is 64 bytes and rustc doesn't consider that big enough to fire the warning (since I followed the compile warnings to fix all the places). I still made the change though, 64 bytes is still _something_...